### ISSUE TITLE
Open link elements in _blank window

### ISSUE THREAD

**Issue by Bartvaderkin** [open-formulieren/open-forms#1420]
Taiga 425

Links created by the form-builder should always open in a new window:

- [x] CKEditor in formio builder
- [x] ~~TinyMCE in the admin forms~~ (was already configurable and default)

We've discussed doing a datamigration for existing forms but have to see if that's practical.

**Comment by Bartvaderkin** [comment]
The content element is built on a textarea element with wysiwyg=True and some settings that will initiate ckeditor with some settings it pulls from its internals options. I noticed the texarea component in the formio builder sandbox has a config option to supply editor settings, and when I do I see them in the component configuration on the right (its disabled in our build).

I've found the ckeditor settings we want to add on the default settings for the content element (see referenced commit/PR)

I tried extending the content component and insert our settings but I had issues making it work.

Some links on what I could find in the formio source:

https://github.com/formio/formio.js/blob/v4.13.0/src/components/_classes/component/Component.js#L2070
https://github.com/formio/formio.js/blob/v4.13.0/src/components/textarea/TextArea.js#L124-L130
https://github.com/formio/formio.js/blob/v4.13.0/src/components/content/Content.form.js
